bundle/LegacyMapper/Configuration.php

```diff
@@ -277,10 +277,21 @@ private function getImageSettings()
     private function getMultiSiteSettings()
     {
         $rootLocationId = $this->configResolver->getParameter( 'content.tree_root.location_id' );
+        $indexPage = $this->configResolver->getParameter( 'index_page' );
         $defaultPage = $this->configResolver->getParameter( 'default_page' );
         if ( $rootLocationId === null )
         {
-            return array();
+            // return SiteSettings if there is no MultiSite (rootLocation is not defined)
+            $result = array();
+            if ( $indexPage )
```

Comment:
I was just thinking about something such as `!== null`, if null is what we might get. We usually don't use very "fuzzy" tests like this, but yes, the value is validated.